docs/documentation/release_notes/topics/26_0_0.adoc

```diff
@@ -130,6 +130,10 @@ not recommended at all in production deployments of Keycloak, it is fairly frequ
 of `localhost`. As an alternative to the `_LEGACY` cookies Keycloak now doesn't set the `secure` flag and sets `SameSite=Lax`
 instead of `SameSite=None` when it detects an insecure context is used.
 
+= Hostname v1 feature removed
+
+The deprecated hostname v1 feature was removed. This feature was deprecated in {project_name} 24 and replaced by the hostname v2 theme. If you are still using this feature, you should migrate to the hostname v2 feature.
```

Comment:
```suggestion
The deprecated `hostname:v1` feature was removed. This feature was deprecated in {project_name} 25 and replaced by the hostname v2 theme. If you are still using this feature, you must migrate to the `hostname:v2` feature.

For more details, see the https://www.keycloak.org/server/hostname[Configuring the hostname (v2)].
```
1. I think it was deprecated in [Keycloak 25](https://www.keycloak.org/2024/06/keycloak-2500-released.html#_new_hostname_options), right? 
2. Maybe formatting like this might be better? As users dealing with the `--features=hostname:vX` syntax?
3. Perhaps it would be good to explicitly mention the guide for hostname v2?

Reply:
> I think it was deprecated in [Keycloak 25](https://www.keycloak.org/2024/06/keycloak-2500-released.html#_new_hostname_options), right?

Yes, sorry this was a cut and paste error from the account console removal.

> Maybe formatting like this might be better? As users dealing with the --features=hostname:vX syntax?

I don't have a strong opinion on that. We on have 4 references in the docs to the hostname: format, all the other references are to hostanme v.

> Perhaps it would be good to explicitly mention the guide for hostname v2?

That's fine. Here again I was just copying the other removal notice - it seemed to assume users were aware from the deprecation notice / warnings.

Reply:
> it seemed to assume users were aware from the deprecation notice / warnings.

@shawkins Our release notes contain a lot of information, and it's very easy to get lost, so I'd be more explicit in this case and mention it again to be sure. Thanks for the update!